okhttp/src/main/java/okhttp3/internal/tls/OkHostnameVerifier.kt

```diff
@@ -70,6 +70,7 @@ object OkHostnameVerifier : HostnameVerifier {
    * @param pattern domain name pattern from certificate. May be a wildcard pattern such as
    *     `*.android.com`.
    */
+  @Suppress("NAME_SHADOWING")
```

Comment:
Can we suppress this warning in our compiler config?

Reply:
Haven't worked that out yet.  Moving away from the code to the top of the file for now.